downloader.py

```diff
@@ -1,49 +1,44 @@
-import hashlib
-import random
 import requests
+import logging
 import time
-import traceback
-from multiprocessing import Process, Manager
+import os
+from threading import Thread
 
-def fetch_images(hashes_seen, filesystem_lock):
-    while True:
-        try:
-            req = requests.get('https://thispersondoesnotexist.com/image', headers={'User-Agent':'fada'})
-            img = req.content
-            if len(img) < 50_000:
-                print(f'!!! corrupted image (size {len(img)}), skipping.')
-                continue
-            hsh = hashlib.sha256(img).hexdigest()
-            with filesystem_lock:
-                if hsh not in hashes_seen:
-                    fileid = int(time.time()) * 10
-                    while fileid in hashes_seen.values():
-                        fileid += 1
-                    hashes_seen[hsh] = fileid
-                    with open(f'download/{fileid}.jpg', 'wb') as fo:
-                        fo.write(img)
+USER_AGENT = 'fada'
+INTERVAL = 1.2
```

Comment:
```suggestion
INTERVAL = 1.0
```

The cache is refreshed every 1.2 seconds, but working on this same interval could cause issues if we happen to have downloads exactly in sync with the refresh, potentially missing a few images.